flang/lib/Lower/IntrinsicCall.cpp

```diff
@@ -800,7 +800,7 @@ IntrinsicLibrary::genIntrinsicCall(llvm::StringRef name, mlir::Type resultType,
 
   auto runtimeCallGenerator = getRuntimeCallGenerator(name, soughtFuncType);
   return genElementalCall(runtimeCallGenerator, name, resultType, args,
-                          /* outline */ true);
+                          /* outline */ false);
```

Comment:
What does this change do?

Reply:
> What does this change do?

I think I found the previous history of discussion on that.
https://github.com/flang-compiler/f18-llvm-project/issues/289#issuecomment-661819554

Reply:
Yes, that is the temporary workaround.